package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "react-decoration",
-  "version": "2.0.0",
+  "name": "react-decoration-nodejs",
```

Comment:
could you please restore the original name of the library?

Reply:
Whoops, sorry! This was an artifact from an internal registry namespacing. I'll fix this.